specutils/fitting/fitmodels.py

```diff
@@ -377,7 +382,7 @@ def fit_lines(spectrum, model, fitter=fitting.LevMarLSQFitter(calc_uncertainties
 
 def _fit_lines(spectrum, model, fitter=fitting.LevMarLSQFitter(calc_uncertainties=True),
                exclude_regions=None, weights=None, window=None, get_fit_info=False,
-               ignore_units=False, **kwargs):
+               ignore_units=False, filter_non_finite=False, **kwargs):
```

Comment:
I think you'll need to add `filter_non_finite` to the `kwargs` in the `LevMarLSQFitter`, looks like tests are failing because other fitters don't expect that keyword argument.

Reply:
ah good point, I thought I caught this by setting it to False when the fitter isn't LevMarLSQFitter but those fitters won't expect the flag at all